docker-compose.yml

```diff
@@ -29,40 +29,20 @@ services:
       context: ./rps
       dockerfile: ./Dockerfile
     ports: 
-      - "${RPS_WEBSOCKETPORT}:${RPS_WEBSOCKETPORT}"
-      - "${RPS_WEB_PORT}:${RPS_WEB_PORT}"
+      - "${RPSWEBSOCKETPORT}:8080"
+      - "${RPSWEBPORT}:8081"
     depends_on: 
       - mps
+    env_file:
+      - .env
     environment: 
```

Comment:
where are the environmental variables being imported with this change?

Reply:
the .env file when you run docker-compose up. You can run `docker-compose config`  to see the configured values